test/unittest/rtps/reader/WriterProxyTests.cpp

```diff
@@ -121,6 +121,7 @@ TEST(WriterProxyTests, MissingChangesUpdate)
     // and lastSN.value < firstSN.value
     bool assert_liveliness = false;
     uint32_t heartbeat_count = 1;
+    int32_t current_sample_lost = 0;
```

Comment:
Take the chance to check the returned values on `current_sample_lost`

Reply:
Done on 048375d098e4696bdad8a2eec13afab1fbc7420c